<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/icons/appointment-new.png</normaloff>:/icons/appointment-new.png</iconset>
  </property>
  <property name="documentMode">
   <bool>true</bool>
  </property>
  <property name="dockNestingEnabled">
   <bool>true</bool>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QMdiArea" name="mdiArea"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <widget class="QMenu" name="menuOpen_Recent">
     <property name="title">
      <string>Open Recent</string>
     </property>
     <addaction name="actionClear_Recent"/>
     <addaction name="separator"/>
    </widget>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="menuOpen_Recent"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="separator"/>
    <addaction name="actionPrint"/>
    <addaction name="actionSave_Image"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="actionDelete"/>
    <addaction name="actionSelect_All"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionContent"/>
    <addaction name="actionIndex"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>View</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_Settings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionPreferences"/>
   </widget>
   <widget class="QMenu" name="menu_Window">
    <property name="title">
     <string>Window</string>
    </property>
    <addaction name="actionFull_screen"/>
    <addaction name="actionMinimize"/>
    <addaction name="actionMaximize"/>
    <addaction name="actionRestore"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Edit"/>
   <addaction name="menu_View"/>
   <addaction name="menu_Settings"/>
   <addaction name="menu_Window"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QToolBar" name="fileToolBar">
   <property name="windowTitle">
    <string>Tools</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
  </widget>
  <widget class="QToolBar" name="editToolBar">
   <property name="windowTitle">
    <string>Tools</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionCut"/>
   <addaction name="actionCopy"/>
   <addaction name="actionPaste"/>
   <addaction name="separator"/>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
  </widget>
  <widget class="QToolBar" name="settingsToolBar">
   <property name="windowTitle">
    <string>Tools</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionProperties"/>
  </widget>
  <widget class="QToolBar" name="helpToolBar">
   <property name="windowTitle">
    <string>Tools</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionContent"/>
   <addaction name="actionIndex"/>
   <addaction name="separator"/>
   <addaction name="actionAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="sizeGripEnabled">
    <bool>true</bool>
   </property>
   <widget class="QSpinBox" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>100</width>
      <height>18</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="showGroupSeparator" stdset="0">
     <bool>true</bool>
    </property>
    <property name="maximum">
     <number>20</number>
    </property>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget">
   <property name="windowTitle">
    <string>A</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QListView" name="documentList">
       <property name="contextMenuPolicy">
        <enum>Qt::CustomContextMenu</enum>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::AllEditTriggers</set>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionRectVisible">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_2">
   <property name="windowIcon">
    <iconset theme="[Theme]window-new">
     <normaloff>../../../../Users/PC/.designer/backup</normaloff>../../../../Users/PC/.designer/backup</iconset>
   </property>
   <property name="windowTitle">
    <string>B</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QTreeView" name="treeView_2"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_3">
   <property name="windowTitle">
    <string>C</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_4">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QTreeView" name="treeView"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_4">
   <property name="windowTitle">
    <string>D</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QListView" name="listView_2"/>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionContent">
   <property name="icon">
    <iconset theme="help-browser" resource="../../data/icons/default/default.qrc">
     <normaloff>:/icons/help-browser</normaloff>:/icons/help-browser</iconset>
   </property>
   <property name="text">
    <string>&amp;Content</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionIndex">
   <property name="icon">
    <iconset theme="x-office-address-book" resource="../../data/icons/default/default.qrc">
     <normaloff>:/icons/x-office-address-book</normaloff>:/icons/x-office-address-book</iconset>
   </property>
   <property name="text">
    <string>&amp;Index</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+F1</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset theme="document-open" resource="../../data/icons/default/default.qrc">
     <normaloff>:/icons/document-open</normaloff>:/icons/document-open</iconset>
   </property>
   <property name="text">
    <string>&amp;Open...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="icon">
    <iconset theme="media-eject" resource="../../data/icons/default/default.qrc">
     <normaloff>:/icons/media-eject</normaloff>:/icons/media-eject</iconset>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F4</string>
   </property>
  </action>
  <action name="actionProperties">
   <property name="icon">
    <iconset theme="preferences-system" resource="../../data/icons/default/default.qrc">
     <normaloff>:/icons/preferences-system</normaloff>:/icons/preferences-system</iconset>
   </property>
   <property name="text">
    <string>Properties...</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset theme="document-new" resource="../../data/icons/default/default.qrc">
     <normaloff>:/icons/document-new</normaloff>:/icons/document-new</iconset>
   </property>
   <property name="text">
    <string>&amp;New..</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset theme="document-save" resource="../../data/icons/default/default.qrc">
     <normaloff>:/icons/document-save</normaloff>:/icons/document-save</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="icon">
    <iconset theme="document-save-as" resource="../../data/icons/default/default.qrc">
     <normaloff>:/icons/document-save-as</normaloff>:/icons/document-save-as</iconset>
   </property>
   <property name="text">
    <string>Save &amp;As...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="icon">
    <iconset theme="document-print" resource="../../data/icons/default/default.qrc">
     <normaloff>:/icons/document-print</normaloff>:/icons/document-print</iconset>
   </property>
   <property name="text">
    <string>&amp;Print...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionSave_Image">
   <property name="text">
    <string>Save Image...</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset theme="application-exit" resource="../../data/icons/default/default.qrc">
     <normaloff>:/icons/application-exit</normaloff>:/icons/application-exit</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Alt+F4</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset theme="edit-undo" resource="../../data/icons/default/default.qrc">
     <normaloff>:/icons/edit-undo</normaloff>:/icons/edit-undo</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset theme="edit-redo" resource="../../data/icons/default/default.qrc">
     <normaloff>:/icons/edit-redo</normaloff>:/icons/edit-redo</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset theme="edit-cut" resource="../../data/icons/default/default.qrc">
     <normaloff>:/icons/edit-cut</normaloff>:/icons/edit-cut</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset theme="edit-copy" resource="../../data/icons/default/default.qrc">
     <normaloff>:/icons/edit-copy</normaloff>:/icons/edit-copy</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset theme="edit-paste" resource="../../data/icons/default/default.qrc">
     <normaloff>:/icons/edit-paste</normaloff>:/icons/edit-paste</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset theme="edit-delete" resource="../../data/icons/default/default.qrc">
     <normaloff>:/icons/edit-delete</normaloff>:/icons/edit-delete</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionSelect_All">
   <property name="icon">
    <iconset theme="select-all" resource="../../data/icons/default/default.qrc">
     <normaloff>:/icons/edit-select-all</normaloff>:/icons/edit-select-all</iconset>
   </property>
   <property name="text">
    <string>Select All</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="icon">
    <iconset theme="preferences-system" resource="../../data/icons/default/default.qrc">
     <normaloff>:/icons/preferences-system</normaloff>:/icons/preferences-system</iconset>
   </property>
   <property name="text">
    <string>Preferences...</string>
   </property>
  </action>
  <action name="actionMinimize">
   <property name="text">
    <string>Minimize</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset theme="system-users" resource="../../data/icons/default/default.qrc">
     <normaloff>:/icons/system-users</normaloff>:/icons/system-users</iconset>
   </property>
   <property name="text">
    <string>About...</string>
   </property>
  </action>
  <action name="actionClear_Recent">
   <property name="text">
    <string>Clear &amp;Recent</string>
   </property>
  </action>
  <action name="actionMaximize">
   <property name="text">
    <string>Maximize</string>
   </property>
  </action>
  <action name="actionFull_screen">
   <property name="text">
    <string>Full screen</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+F11</string>
   </property>
  </action>
  <action name="actionRestore">
   <property name="text">
    <string>Restore</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../data/icons/default/default.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionMinimize</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>showMinimized()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>383</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionMaximize</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>showMaximized()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>383</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionFull_screen</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>showFullScreen()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>383</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionRestore</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>showNormal()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>383</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
